tee-worker/ts-tests/integration-tests/di_vc.test.ts

```diff
@@ -13,33 +13,13 @@ import type { IntegrationTestContext } from './common/type-definitions';
 import { aesKey } from './common/call';
 import { LitentryPrimitivesIdentity } from 'sidechain-api';
 import { subscribeToEventsWithExtHash } from './common/transactions';
+import { basicAssertions, extraAssertions } from './common/utils/vc-helper';
+
```

Comment:
The names `basicAssertions` and `extraAssertions` aren't telling me much about what these are _supposed_ to be or why they are split that way; could you give them more descriptive names please?

Reply:
You probably need to re-request the review, otherwise @grumpygreenguy won't know it's ready to review :)